src/main/java/com/example/repository/AdministratorRepository.java

```diff
@@ -75,11 +75,9 @@ public Administrator findByMailAddressAndPassward(String mailAddress, String pas
 	 * @param administrator 管理者情報
 	 */
 	public void insert(Administrator administrator) 
-	{
-	SqlParameterSource param = new BeanPropertySqlParameterSource(administrator);
+	{SqlParameterSource param = new BeanPropertySqlParameterSource(administrator);
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def lcurly' has incorrect indentation level 8, expected level should be 4.